sceptre/synapseprod/config/prod/snowflake-access.yaml

```diff
@@ -1,6 +1,8 @@
 template:
-  path: snowflake-access.yaml
+  type: http
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.7.8/IAM/snowflake-synapse-access.yaml
```

Comment:
where is `snowflake-synapse-access.yaml`? i only see `templates/IAM/snowflake-bucket-access.yaml` in PR https://github.com/Sage-Bionetworks/aws-infra/pull/398